Makefile

```diff
@@ -46,4 +56,12 @@ install-docker-%: bin/docker-%
 	cp $< ~/.docker/cli-plugins/
 
 .PHONY: install-docker-plugins
-install-docker-plugins: $(addprefix install-,$(DOCKER_PLUGINS))
+install-docker-plugins: $(addprefix install-,$(DOCKER_PLUGINS)) ## installs the docker plugins
+	cp $(addprefix bin/,$(DOCKER_PLUGINS)) ~/.docker/cli-plugins/
+
+lint: ## use goimports to lint
+	$(GOPATH)/bin/goimports -d -e -local github.com/notaryproject/nv2 $$(go list -f {{.Dir}} ./...)
+
+install-tools: download ## install tools from tools.go
```

Comment:
based on the above comment. We don't need to install tools if we use `go fmt`.

Reply:
goimports also takes care of ordering the imports. So they are nicely ordered. That gives some consistency as well on the imports reducing diffs in future code changes.